monkey/monkey_island/cc/services/ransomware_report.py

```diff
@@ -0,0 +1,13 @@
+from monkey_island.cc.services.reporting.report import ReportService
+
+
+class RansomwareReportService:
```

Comment:
I realize this is the pattern for the rest of the services, but a class that stores no state and only contains static methods doesn't make a whole lot of sense. The class doesn't serve any purpose. This pattern seems like something someone picked up from Java and carried over to Python.

We can just make `get_exploitation_details()` a function and call `ransomware_report.get_exploitation_details()`